File: 프로그래머스/unrated/181927. 마지막 두 원소/마지막 두 원소.java
class Solution {
    public int[] solution(int[] num_list) {
        int len = num_list.length;
        int[] answer= new int[len + 1];
        for (int i = 0; i < len; i++)
            answer[i] = num_list[i];
        answer[len] = answer[len - 2] < answer[len - 1] ? answer[len - 1] - answer[len - 2] : answer[len - 1] * 2;
        return answer;
    }
}

File: 프로그래머스/Lv.0/181927. 마지막 두 원소/마지막 두 원소.js
function solution(num_list) {
    var end = num_list.length - 1;
    var ans = num_list.slice(0);
    ans.push(ans[end - 1] < ans[end] ? (ans[end] - ans[end - 1]) : ans[end] * 2);
    return ans;
}